app/models/profile.rb

```diff
@@ -17,6 +17,12 @@ class Profile < ApplicationRecord
 
   delegate :email, to: :user
 
+  scope :search_profiles, (lambda do |search_query|
```

Comment:
Remove unneeded parentheses around `lambda` block
Also rename it to `search` (`scope :search, lambda do |query|.....`)